src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -54,24 +54,45 @@ public class CameraData
         private readonly Vector3D defaultCameraUpDirection = new Vector3D(0, 1, 0);
         private const double defaultNearPlaneDistance = 0.1;
         private const double defaultFarPlaneDistance = 10000000;
-         
-        public Point3D EyePosition { get; set; }
-        public Vector3D UpDirection { get; set; }
-        public Vector3D LookDirection { get; set; }
-        public string Name { get; set; }
+
+        [JsonIgnore]
+        public Point3D EyePosition { get { return new Point3D(EyeX, EyeY, EyeZ); } }
+        [JsonIgnore]
+        public Vector3D UpDirection { get { return new Vector3D(UpX, UpY, UpZ); } }
+        [JsonIgnore]
+        public Vector3D LookDirection { get { return new Vector3D(LookX, LookY, LookZ); } }
         [JsonIgnore]
         public double NearPlaneDistance { get; set; }
         [JsonIgnore]
         public double FarPlaneDistance { get; set; }
 
+        // JSON camera data
+        public string Name { get; set; }
+        public double EyeX { get; set; }
+        public double EyeY { get; set; }
+        public double EyeZ { get; set; }
+        public double LookX { get; set; }
+        public double LookY { get; set; }
+        public double LookZ { get; set; }
+        public double UpX { get; set; }
+        public double UpY { get; set; }
+        public double UpZ { get; set; }
```

Comment:
Looks like you are changing what got serialized in the graph,Json. Can you put an updated Camera block in the PR description?
E.g. Right now it's
```json
"Cameras": {
      "EyePosition": "-17,24,50",
      "UpDirection": "0,1,0",
      "LookDirection": "12,-13,-58",
      "Name": "Background Preview"
    }
```
After the change it will be 
```json
"Cameras": {
      "EyeX": "-17",
      "EyeY": "24",
       ...
      "Name": "Background Preview"
    }
```